docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,426 @@
+
+To run another validator, you will need the following command in another terminal:
+
+```
+./target/release/polkadot --chain /rococo-#.json --tmp --ws-port 9966 --port 30335 --NODE --bootnodes /ip4/<Alice ID>/tcp/30333/p2p/<Alice peer ID>
+```
+
+> Important note: When you run another validator, change the values for `--ws-port` and `--port`, so
+> that they aren't conflicting with eachother. The Rococo `json` file should be kept the same here.
+
+## Connect to Polkadot JS UI
+
+At this point, you are now ready to connect your local node to the Polkadot JS UI. To do so, head
+over to the [user interface](https://polkadot.js.org/apps/) and change the network to `Local Node`.
+For now, you can change the custom endpoint to one of your validator `--ws-port` values. When this
+the network is connected, the main page should look like this:
+
+![rococo-connected-to-ui](assets/rococo/rococo-ui-connected.png)
+
+## Starting the Parachain with the Relay Chain
+
+In order for the parachain to launch properly, we first need to register it with the Relay Chain. We
+will begin by running this command to generate a WASM file inside your substrate parachain template
+folder:
+
+```
+./target/release/parachain-collator export-genesis-wasm > para-wasm
+```
+
+Next, inside the same terminal, you will need to run the collator node.
+
+```
+./target/release/parachain-collator --tmp --validator --ws-port # --port # --parachain-id 200 -- --chain <Path to>/rococo-#.json --bootnodes /ip4/<Alice IP>/tcp/<Alice port>/p2p/<Alice Peer ID> <Other Relay Chain Node(s)>
+```
+
+> Tip: to check what your validator nodes are doing on the network so far, you can look for:
+> `✍️ Starting parachain attestation session (parent: 0xe482…3e12) with active duty LocalDuty { validation: Relay, index: 0 `
+> in your terminal. In this case, this validator node is working on the Relay Chain.
+
+Note that for `--port` and `--ws-port` you need to set to different values than your validator
+nodes. Also take note that for creating more parachains, you'd need to change the value of the
+`--parachain-id`, and it is not required that the parachain id in this case be `200`, it can be
+whatever id you'd like that to be. The `<Alice port>` should also be replaced with alice's port
+number created.
+
+Once that starts running in your terminal, an important thing to grab is the genesis state. It will
+be in the format:
+
+```
+2020-08-11 13:58:06 Parachain genesis state: 0x00000000000000000000000000000000000000000000000000000000000000000097600fcfeeed0c7c2e7d922081a466c4c00f2af96ce17f4a07d59e7d47e8354b03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
+```
+
+This is the genesis state of the parachain and will be used to actually register the parachain in
+the Polkadot JS UI. It is recommended to keep this in a safe place to refer back to.
+
+At this point the console log should resemble something like this:
+
+```2020-08-25 13:32:11 Parachain Collator Template
```

Comment:
```suggestion
```
2020-08-25 13:32:11 Parachain Collator Template
```

Reply:
I'm a bit confused on why you want this deleted. Could you explain?

Reply:
Oops, this didn't get formatted right. I was suggesting a newline after the code fence:

![image](https://user-images.githubusercontent.com/25539605/91725029-f8120e80-eb52-11ea-9ce7-25168e306b61.png)

Reply:
OH okay cool, I can add this in :)